tvos-types.d.ts

```diff
@@ -112,7 +112,7 @@ declare module 'react-native' {
 
   export type FocusGuideMethods = {
     setDestinations: (
-      destinations: (?React.ElementRef<HostComponent<unknown>>)[],
+      destinations: (React.ElementRef<HostComponent<unknown>> | null)[],
```

Comment:
Nullability is pretty much a convenience thing. React refs are nullable by default and users will be passing refs as destinations. We let destinations nullable and filter out null values under the hood so devs won't have to place null checks everywhere

Reply:
```suggestion
      destinations: (React.ElementRef<HostComponent<unknown>> | null | undefined)[],
```